### PR TITLE
Initialize a Foundry project

### PR DESCRIPTION
**Pull request by LOUDINISouad**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: test
```
Comment (achab) [inline .github/workflows/test.yml:1]: @LOUDINISouad do you understand what this file does?

File: script/Counter.s.sol [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: UNLICENSED
```
[the remark this answers is not in the document]
Comment (achab) [reply script/Counter.s.sol:1]: ok great, but what do we mean by initialization? you can read the first paragraph of this reference: https://book.getfoundry.sh/tutorials/solidity-scripting

File: test/Hello_world.sol [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: UNLICENSED
```
Comment (achab) [inline test/Hello_world.sol:1]: this file is not in the right place: it is a contract file, not a test file.
- it does not inherit from `Test` like `CounterTest` (see https://github.com/Amenokal-Labs/decentralized-exchange/pull/3/files#diff-8c7a37ff0156acc4a48ceac17fb6d33836dab5834df4064c2a7e5d08c8650662R7)
- extension of tests is `.t.sol`, and not `.sol`

File: README.md [hunk 4]
```diff
@@ -1 +1,18 @@
-# decentralized-exchange
\ No newline at end of file
+# decentralized-exchange
```
Comment (achab) [inline README.md:1]: very good readme

File: .github/workflows/test.yml (hunk 1 above)
Comment (achab) [reply .github/workflows/test.yml:1]: @LOUDINISouad so?

**Comment by achab** [comment]
Hi @LOUDINISouad, can you give the PR a more descriptive name? Like "Initialize a Foundry project"

File: package.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "decentralized-exchange",
+  "version": "1.0.0",
```
Comment (LOUDINISouad) [inline package.json:3]: this is the version

File: .github/workflows/test.yml (hunk 1 above)
Comment (achab) [reply .github/workflows/test.yml:1]: ok, that's a good summary. github action is a tool to run CI/CD process on github servers. we will add it this week in a another PR.

**Review by achab — APPROVED** [review]
Very good PR. Please remove the github action file, and then you can merge the PR

**Pull request merged**